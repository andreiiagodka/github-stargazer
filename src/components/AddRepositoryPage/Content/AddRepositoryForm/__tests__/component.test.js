import { shallow } from 'enzyme'
import configureStore from 'redux-mock-store'

import AddRepositoryFormComponent from '..'

describe('<AddRepositoryFormComponent />', () => {
  it('renders correctly', () => {
    const mockStore = configureStore([])
    const store = mockStore({})

    const wrapper = shallow(<AddRepositoryFormComponent.WrappedComponent store={store} />)

    expect(wrapper).toMatchSnapshot()
  })
})
