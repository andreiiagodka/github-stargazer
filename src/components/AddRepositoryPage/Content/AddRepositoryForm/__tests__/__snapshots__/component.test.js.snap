// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AddRepositoryFormComponent /> renders correctly 1`] = `
<Formik
  displayName="AddRepositoryFormContainer"
  handleSubmit={[Function]}
  initialValues={
    Object {
      "name": "",
    }
  }
  onSubmit={[Function]}
  store={
    Object {
      "clearActions": [Function],
      "dispatch": [Function],
      "getActions": [Function],
      "getState": [Function],
      "replaceReducer": [Function],
      "subscribe": [Function],
    }
  }
  validationSchema={[Function]}
>
  <Component />
</Formik>
`;
